tethys_apps/cli/scaffold_templates/app_templates/default/tethysapp/+project+/templates/+project+/base.html_tmpl

```diff
@@ -38,5 +38,6 @@
 
 {% templatetag openblock %} block scripts {% templatetag closeblock %}
   {% templatetag openvariable %} block.super }{{ "}" }}
+  {% include ”{{project}}/analytics.html" %}
```

Comment:
Check the first quote character on this line. It renders differently than the other one.

Reply:
good catch. I commited a change to fix that.